script.js

```diff
@@ -1,406 +1,193 @@
-      '2 celery stalks, sliced',
-      '2.5 pounds organic beef stew meat, cubed',
-      '2 tablespoons tomato paste',
-      '5 cloves garlic',
-      '2 bay leaves',
-      '3 sprigs thyme',
-      '3 sprigs Italian parsley',
-      '1/2 teaspoon black peppercorns'
-    ],
-    source: 'Food52',
-    totalTime: 60,
-    url: 'https://food52.com/recipes/3712-meat-stock',
-    image: './recipe-images/meat.jpg'
-  },
-  {
-    name: 'Homemade Meat Broth',
-    cuisineType: 'american',
-    ingredients: [
-      '1 teaspoon salt',
-      '1 carrot, peeled',
-      '1 medium onion, peeled',
-      '2 or 3 celery stalks',
-      '¼ red or yellow bell pepper, stripped of all its seeds',
-      '1 small boiling potato, peeled',
-      '1 ripe, fresh tomato, or 1 canned Italian plum tomato, drained of juice',
-      '5 pounds assorted pieces of meat and bones (see meat suggestions above), of which no less than 1½ pounds is all meat'
-    ],
-    source: 'Cookstr',
-    totalTime: 60,
-    url: 'http://www.cookstr.com/recipes/il-brodo-homemade-meat-broth',
-    image: './recipe-images/meat.jpg'
-  },
-  {
-    name: 'Spice-Rubbed Grilled Flap Meat (Sirloin Tip) Recipe',
-    cuisineType: 'south-american',
-    ingredients: [
-      '1 tablespoon whole black peppercorns, toasted',
-      '1 teaspoon coriander seed, toasted',
-      '1 teaspoon fennel seed, toasted',
-      '1 teaspoon cumin pods, toasted',
-      '1 teaspoon red pepper flakes',
-      '1/2 teaspoon dried oregano',
-      '2 medium cloves garlic, minced (about 2 teaspoons)',
-      '2 tablespoons vegetable or canola oil',
-      '1 whole flap meat steak, 2 to 2 1/2 pounds',
-      'Kosher salt'
-    ],
-    source: 'Serious Eats',
-    totalTime: 240,
-    url: 'http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html',
-    image: './recipe-images/grilled.jpg'
+// Function to display movies (generic to handle sorting and filtering)
+function displayMovies(movieList) {
+  const moviesSection = document.getElementById('movies-section');
+  moviesSection.innerHTML = ''; // Clear previous movies
+
+  movieList.forEach(movie => {
+    const movieElement = `
+      <article>
+        <img src="${movie.image}" alt="${movie.title}">
+        <div class="card-content">
+          <h2>${movie.title}</h2>
+          <p><strong>Director:</strong> ${movie.director} | <strong>Year:</strong> ${movie.year}</p>
+          <p><strong>Genre:</strong> ${movie.genre}</p>
+          <p class="description">${movie.description}</p>
+        </div>
+      </article>`;
+    moviesSection.innerHTML += movieElement;
+  });
+}
+
+// Function to display all movies
+function displayAllMovies() {
+  displayMovies(movies);
+}
+
+// Function to filter and display movies based on genre
+function recommendMovieByGenre(genre) {
+  const filteredMovies = movies.filter(movie => {
+    const genres = movie.genre.split(',').map(g => g.trim().toLowerCase());
```

Comment:
Suggestion to have a button to be able to see all the movies in the array /Kasia, Cholpon, Nella